buildSrc/src/main/kotlin/io.customer/android/Versions.kt

```diff
@@ -2,6 +2,9 @@ package io.customer.android
 
 object Versions {
     internal const val ANDROID_GRADLE_PLUGIN = "7.2.0"
+
+    // Using alpha version as current stable version (1.0.0) has minSdkVersion 23
+    internal const val ANDROIDX_SECURITY_CRYPTO = "1.1.0-alpha06"
```

Comment:
Because 1.1-alpha releases support minSdk 21 while 1.0 supports minSdk 23, do we anticipate that 1.1 stable will move to minSdk 21? 

I would like to avoid us having to bump our minSdk because of a dependency.